build_iso.sh

```diff
@@ -20,12 +20,123 @@ docker volume rm build_debian_iso_ext 2>/dev/null
 
 set -ex
 
+if [ ! -f "$wd"/etc_fai/grub.cfg ]; then
+  cp "$wd"/etc_fai/grub_base.cfg "$wd"/etc_fai/grub.cfg
+fi
+
+finalClasses="SEAPATH_CLUSTER,SEAPATH_DBG,SEAPATH_KERBEROS,SEAPATH_COCKPIT,"
+
+if [ "$1" == "--custom" ]; then
+
+  cp "$wd"/etc_fai/grub_base.cfg "$wd"/etc_fai/grub.cfg
+  # add index to an array, for each value
+  function addIndexToArray1 {
+    local -n myarray=$1
+    for idx in $(seq 1 ${#myarray[@]}); do
+      arrVar+=("$idx"")")
+      arrVar+=("${myarray[$((idx-1))]}")
+    done
+  }
+  # add index to an array, every 2 values
+  function addIndexToArray2 {
+    local -n myarray=$1
+    for idx in $(seq 1 $((${#myarray[@]}/2))); do
+      arrVar+=("$idx"")")
+      arrVar+=("${myarray[$((idx*2-2))]}")
+      arrVar+=("${myarray[$((idx*2-1))]}")
+    done
+  }
+
+  listClasses=("SEAPATH_CLUSTER" "ON" "SEAPATH_DBG" "ON" "SEAPATH_COCKPIT" "ON" "SEAPATH_KERBEROS" "ON")
+  arrVar=()
+  addIndexToArray2 listClasses
+
+  CHOICES=$(whiptail --separate-output --checklist "Choose package classes to add to iso" 18 60 7 \
```

Comment:
That look great, but I was not able to select (or unselect) any entry with the "enter" key. When I press enter, it understands it as a "ok" and goes to the next step.
Do I need to tap a specific key (in case it should be mentionned in the README)
Or is it a bug ?

Reply:
This is related to whiptail which is pretty standard: the select/unselect is the space bar.